src/mlpack/tests/softmax_regression_test.cpp

```diff
@@ -277,37 +283,38 @@ TEST_CASE("SoftmaxRegressionMultipleClasses", "[SoftmaxRegressionTest]")
   GaussianDistribution g4(arma::vec("4.0 1.0 1.0 2.0 7.0"), identity);
   GaussianDistribution g5(arma::vec("1.0 0.0 1.0 8.0 3.0"), identity);
```

Comment:
We could have initialised it with the same element type of MatType so we could avoid the conv later but it is fine no need to do anything

Reply:
Yeah, the issue here is that `GaussianDistribution` is specific to `arma::mat`, and I didn't want to take the time in this PR to do that too... it would make it even bigger than it already is.  I'll definitely do that before the end of this documentation saga, and will come back here to clean this up then. :+1: